src/transaction.ts

```diff
@@ -651,8 +651,7 @@ export class Snapshot extends EventEmitter {
    * **Performance Considerations:**
    *
    * This method wraps the streaming method,
-   * {@link Snapshot#run} for your convenience. All rows will
-   * be stored in memory before being released to your callback. If you intend
+   * {@link Snapshot#run} for your convenience. All rows are stored in memory before being released to your callback. If you intend
```

Comment:
We try to limit docs and code to 80 chars per line. This looks like it went over